src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,29 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+    this.hidden = false;
```

Comment:
The 'hidden' property is initialized in the Animal class, but it is only relevant to Herbivores. Consider initializing this property within the Herbivore class instead to avoid unnecessary properties on Carnivore instances and to better align with the principle of class responsibility.